Week-9/demo-react-app/src/Home.js

```diff
@@ -0,0 +1,59 @@
+import { useState, useEffect } from "react";
+import './Home.css'
+
+function Note(props){
+
+    function handleCheck(id){
+        var col = document.getElementById(id), check = document.getElementById(id+"_done")
+        if(check.checked)
+            col.style.textDecoration = "line-through";
+        else
+            col.style.textDecoration = "initial";
+    }
+
+    return (
+        <div id={props.id} className="note">
+            <input id={`${props.id}_done`} type="checkbox" onChange={() => handleCheck(props.id)}/>
+            <p>{props.noteText}</p>
+            {/* <button onClick={() => removeNote(props.index)}>Delete</button> */}
+        </div>
+    );
+}
+
+function Header() {
+    return (
+        <header>
+            <h1 id="todoAppText">TodoApp</h1>
+        </header>
+    );
+}
+
+export default function Home(){
+
+    const [note,setNote] = useState({
+        description: "",
+        done: false
+    })
+    const [notes,setNotes] = useState([])
+
+    const addNote = () => {
+        setNotes([...notes,note])
+        setNote({description: "",done:false})
+    }
+
+    return (
+        <>
+            <Header/>
+            <div>
+                <input value={note['description']} type="text" placeholder="Enter New Note Here" name="note" onChange={(e) => setNote({description: e.target.value, done:false})}/>
+                <button onClick={addNote}>Add</button>
+            </div>
+            <div className="notes">
+                {notes.map((note,index) => {
+                    return <Note id={`note_${index}`} noteText={note['description']} index={index}/>
+                })}
```

Comment:
Is index prop used plz check

Reply:
Yes this is used in the delete function for note.